versions.txt

```diff
@@ -6,4 +6,7 @@ operator=1.2.0
 
 # Represents the current release of ADOT language instrumentation.
 aws-otel-java-instrumentation=v1.32.1
-aws-otel-python-instrumentation=v0.0.1
\ No newline at end of file
+aws-otel-python-instrumentation=v0.0.1
```

Comment:
out of scope but probably should be updated

Reply:
sure